src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/SoulboundRune.java

```diff
@@ -0,0 +1,104 @@
+package me.mrCookieSlime.Slimefun.Objects.SlimefunItem.items;
+
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.Location;
+import org.bukkit.Sound;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.EntityType;
+import org.bukkit.entity.Item;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.Lists.SlimefunItems;
+import me.mrCookieSlime.Slimefun.Objects.Category;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SimpleSlimefunItem;
+import me.mrCookieSlime.Slimefun.Objects.handlers.ItemDropHandler;
+import me.mrCookieSlime.Slimefun.Setup.Messages;
+import me.mrCookieSlime.Slimefun.Setup.SlimefunManager;
+import me.mrCookieSlime.Slimefun.SlimefunPlugin;
+import me.mrCookieSlime.Slimefun.api.Slimefun;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+public class SoulboundRune extends SimpleSlimefunItem<ItemDropHandler> {
+
+    public SoulboundRune(Category category, ItemStack item, String id, RecipeType type, ItemStack[] recipe) {
+        super(category, item, id, type, recipe);
+    }
+
+    @Override
+    public ItemDropHandler getItemHandler() {
+        return (e, p, i) -> {
+            ItemStack item = i.getItemStack();
+            if (SlimefunManager.isItemSimiliar(item, SlimefunItems.RUNE_SOULBOUND, true)) {
+                if (!Slimefun.hasUnlocked(p, SlimefunItems.RUNE_SOULBOUND, false)) {
+                    Messages.local.sendTranslation(p, "messages.not-researched", true);
+                    return true;
+                }
+
+                Bukkit.getScheduler().scheduleSyncDelayedTask(SlimefunPlugin.instance, () -> {
+                    // Being sure the entity is still valid and not picked up or whatsoever.
+                    if (!i.isValid()) return;
+
+                    Location l = i.getLocation();
+                    Collection<Entity> entites = l.getWorld().getNearbyEntities(l, 1.5, 1.5, 1.5,
+                            entity -> entity.getType() == EntityType.DROPPED_ITEM && entity instanceof Item &&
```

Comment:
You can remove the entity.getType - check, entity instanceof Item should replace that.